string arm_action_caller_id
youbot_motion_interface/Action_Info arm_action
string base_action_caller_id
youbot_motion_interface/Action_Info base_action
